modules/ROOT/pages/getting.started.lake.adoc

```diff
@@ -0,0 +1,229 @@
+= Getting Started with VantageCloud Lake 
```

Comment:
I think the file name should be `getting-started-with-vantagecloud-lake.adoc` to improve google indexing